test/scripts/e2e_subs/app-inner-calls-csp.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
```

Comment:
@algoidurovic Unrelated to the line, but as a friendly reminder for you and me:
* Indexer runs a sanity check test using the output of go-algorand E2E tests.  The process requires some manual steps to Indexer to use tests added here.
* I take the action item to document the process.  I'll open a PR shortly to explain what's involved.
* And then the action on your side will be to test drive the docs upon merging here.

Reply:
Since the PRs were merged, I'm resolving the thread.  I'll leave a friendly reminder in the approval note.